public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+openapi: 3.0.0
```

Comment:
As per [Confluence](https://confluence.tools.tax.service.gov.uk/display/DTRG/OpenAPI+Specification+structure+and+style) guidelines, OpenAPI version number should be `3.0.3`.